src/main/java/org/jenkinsci/test/acceptance/po/LdapSecurityRealm.java

```diff
@@ -16,34 +17,91 @@
 
     private GlobalSecurityConfig context;
 
-    protected final Control ldapServer = control("server");
-    protected final Control advanced = control("advanced-button");
-    protected final Control rootDn = control("rootDN");
-    protected final Control managerDn = control("managerDN");
-    protected final Control managerPassword = control("managerPasswordSecret"/* >= 1.9*/, "managerPassword");
-    protected final Control userSearchBase = control("userSearchBase");
-    protected final Control userSearchFilter = control("userSearch");
-    protected final Control groupSearchBase = control("groupSearchBase");
-    protected final Control groupSearchFilter = control("groupSearchFilter");
+    protected Control ldapServer;
+    protected Control advanced;
+    protected Control advancedServer;
+    protected Control rootDn;
+    protected Control managerDn;
+    protected Control managerPassword;
+    protected Control userSearchBase;
+    protected Control userSearchFilter;
+    protected Control groupSearchBase;
+    protected Control groupSearchFilter;
     /**
      * only available prior ldap plugin version 1.10
      */
-    protected final Control groupMembershipFilter = control("groupMembershipFilter");
-    protected final Control disableLdapEmailResolver = control("disableMailAddressResolver");
-    protected final Control enableCache = control("cache");
-    protected final Control addEnvVariableButton = control("repeatable-add");
+    protected Control groupMembershipFilter;
+    protected Control disableLdapEmailResolver;
+    protected Control enableCache;
+    protected Control addEnvVariableButton;
     /**
      * since version 1.8
      */
-    private final Control displayNameAttributeName = control("displayNameAttributeName");
+    private Control displayNameAttributeName;
     /**
      * since version 1.8
      */
-    private final Control mailAddressAttributeName = control("mailAddressAttributeName");
+    private Control mailAddressAttributeName;
+
+    /**
+     * For changes in GUI in version 1.16
+     */
+    private final boolean version116 = isVersionEqualsOrGreater116();
 
     public LdapSecurityRealm(GlobalSecurityConfig context, String path) {
         super(context, path);
         this.context = context;
+        initControls();
+    }
+
+    private boolean isVersionEqualsOrGreater116() {
+        boolean isVersion116 = true;
+        try {
+            control("configurations/server").resolve();
+        } catch (NoSuchElementException e) {
+            isVersion116 = false;
+        }
+
+        return isVersion116;
+    }
+
+    /*
+     * Controls for configureSecurity page must be setted depending on the current version of ldap plugins due to changes in GUI
```

Comment:
Why, to discover the version first? Existing approach is to specify new and old paths as arguments to `control` so the first one existing wins. just like for `managerPassword` does it.